src/components/Table/TableRow.tsx

```diff
@@ -9,7 +8,7 @@ import { Checkbox, CheckboxProps } from '../Checkbox';
  * If a `render` function is available, use it to get the display value.
  */
 const renderField = <T extends {}>(row: T, column: TableColumn<T>): any => {
-	const value = get(row, column.field);
+	const value = row[column.field];
```

Comment:
I looked in the ui and it seems like the `column.field` is always a straight property name - if we do want to support eg `a.b.c` I think it would be better to do it explicitly rather than accidentally